tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java

```diff
@@ -161,11 +171,35 @@ public void beginTask( String taskName, int totalWork )
                         {
                             final List<Problem> problems = m.findProblems( location.toFile(), override );
 
-                            problems.addAll( problems );
+                            allProblems.addAll( problems );
                         }
                     }
 
-                    m.reportProblems( allProblems, Migration.DETAIL_LONG, "ide" );
+                    if( allProblems.size() > 0 )
+                    {
+                        CodeMigrationProblems cp = new CodeMigrationProblems();
```

Comment:
What happens if there are already Migration Problems in the settings?  Will this overwrite those?